### ISSUE TITLE
Video display delayed

### ISSUE THREAD

**Issue by bgrozev** [jitsi/jitsi-meet#274]
Observed an instance where, for a very short period of time, right after a participant joins, videobridge receives media packets before it is ready to decrypt them. As a result the initial keyframe is not received correctly and this adds additional delay before the video is displayed in the client.

**Comment by Zalmoxisus** [comment]
It always happens for new participants with no webcam or if they denied access to webcam. In this case such a participant receives only audio of other participant, and only after 1:30 - 2 minutes receives (sees) video stream.

**Comment by bgrozev** [comment]
@Zalmoxisus this seems like a separate problem. This one just delays the video a bit (1-2 RTTs I guess). Could you open a new issue for what you observe, please?

**Comment by Zalmoxisus** [comment]
#bgrozev, ok. Thanks!

**Comment by gpolitis** [comment]
This could be a bug in the WebRTC engine, like not requesting PLIs for receiving channels if audio/video is disabled. You could try to disable bundle and rtcp-mux and see if it changes anything.

**Comment by bgrozev** [comment]
To clarify/update: I observed an RTP packet being dropped here:
https://github.com/jitsi/libjitsi/blob/master/src/org/jitsi/impl/neomedia/transform/dtls/DtlsPacketTransformer.java#L799

It was the first RTP packet after the DTLS handshake. The DTLS handshake completed successfully, and the very next packet (another RTP packet) was processed correctly.

**Comment by bgrozev** [comment]
This was fixed in videobridge.
